src/utils/__tests__/dom-test.js

```diff
@@ -0,0 +1,23 @@
+import scrollIntoViewIfNeeded from 'scroll-into-view-if-needed';
+import { scrollIntoView } from '../dom';
+
+jest.mock('scroll-into-view-if-needed');
+
+describe('util/dom', () => {
+    describe('scrollIntoView()', () => {
+        beforeEach(() => {
+            // Set up a place to mount
+            document.body.innerHTML = '<div class="modal"> <div class="button" /> </div>';
+        });
+
+        afterEach(() => {
+            document.body.innerHTML = '';
+        });
+
+        test('should call scrollIntoView', () => {
+            const itemEl = document.querySelector('.button');
+            scrollIntoView(itemEl);
+            expect(scrollIntoViewIfNeeded).toHaveBeenCalled();
```

Comment:
toHaveBeenCalledWith(...) add the args to see if the function is properly passing in stuff.
Maybe add a test when parent is evaluated as null